docs/open-source/code-of-conduct.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Code of conduct
+description: TODO
```

Comment:
This needs to be done before the folder goes live in 2-3 weeks. Idk if we want to do it now, or later?

Reply:
we could fill it now: "The Qiskit Code of Conduct defines and enforces the values and conduct of contributors and participants in the Qiskit open source community."